### PR TITLE
6 add new categories

### PR DESCRIPTION
**Pull request by mayur-ranium**

### CODE REVIEW

File: src/Router/index.js [hunk 1]
```diff
@@ -18,7 +18,18 @@ const router = createRouter({
             path: "/register",
             component: () => import("../views/Register.vue")
 
+        },
+        {
+            path: "/categoryCreate",
```
Comment (abbasali) [inline src/Router/index.js:23]: As mentioned the other day the path should be `/category/create`

File: src/views/CategoryCreate.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            if(categoryName.value == ''){
+                alert("Please Enter Category Name.")
+                return;
+            }
+            const db = getFirestore();
+            const docRef =  addDoc(collection(db, "categories"), {
+            category : categoryName.value,
+        });  
+            categoryName.value = ""
+            alert("Category created succefully.");
```
Comment (abbasali) [inline src/views/CategoryCreate.vue:33]: Use a Toast plugin like https://github.com/ankurk91/vue-toast-notification to show success/error alerts and notifications.

**Review by abbasali — CHANGES_REQUESTED** [review]

File: src/Router/index.js (hunk 1 above)
Comment (mayur-ranium) [reply src/Router/index.js]: done sir

File: src/views/CategoryCreate.vue (hunk 2 above)
Comment (mayur-ranium) [reply src/views/CategoryCreate.vue]: done sir.

**Review by abbasali — APPROVED** [review]

**Pull request merged**